app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -419,8 +444,52 @@ private void createInfoLayout() {
 
         pokeAdapter = new PokemonSpinnerAdapter(this, R.layout.spinner_pokemon, pokeCalculator.pokedex);
         pokemonList.setAdapter(pokeAdapter);
+
+        pokeEvolutionAdapter = new PokemonSpinnerAdapter(this, R.layout.spinner_pokemon, new ArrayList<Pokemon>());
+        extendedEvolutionSpinner.setAdapter(pokeEvolutionAdapter);
+
+        expandedLevelSeekbar.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener() {
+            @Override
+            public void onProgressChanged(SeekBar seekBar, int i, boolean b) {
+                updateCostFields(IVScanResult.scanContainer.oneScanAgo);
+            }
+
+            @Override
+            public void onStartTrackingTouch(SeekBar seekBar) {
+            }
+
+            @Override
+            public void onStopTrackingTouch(SeekBar seekBar) {
+            }
```

Comment:
Can we drop these empty handlers (onStartTrackingTouch, onStopTrackingTouch) in the end, or do we still need them? If so, maybe a comment explaining why would be good.